instrumentation/kafka/kafka-connect-2.6/javaagent/README.md

```diff
@@ -0,0 +1,101 @@
+# Kafka Connect SinkTask Integration Test
```

Comment:
We don't typically add readme's for test classes. A readme that describes the instrumentation would be more useful, if you want to take a look at how some of the other instrumentations are documented